docs/video-miners/getting-started/activation.md

```diff
@@ -1,185 +1,178 @@
-   [explorer](https://explorer.livepeer.org/).
-
-9. Wait for transactions to confirm. You should see the logs of your
-   orchestrator indicating transactions being submitted and confirmed on-chain.
-
-   After the transactions have confirmed, your orchestrator will join the active
-   set in the following round.
-
+---
+title: Activation
+sidebar_position: 4
+---
+
+This guide provides steps to activate on the Livepeer network so that you can receive video streams from broadcasters in order to provide transcoding services. You will set several parameters, including but not limited to the onchain price and address broadcasters use to find an orchestrator/node.
+
+- You will run `livepeer` with an `ethURL` that tells the `livepeer` process how to communicate to the chain.
+
+- Set several parameters via an onchain transaction (i.e., price / pixel, fee cut, reward cut, service URI, etc) required to activate.
+
+> **Note:** Be sure to follow the [Livepeer Quickstart](/video-miners/getting-started/) before proceeding.
+
+## Preparation 
+
+Before Activating:
+- You will identify the GPUs you will be using for receiving streams and transcoding
+- Set your parameters
+- Start your node
+
+### Print a list of your accessible GPUs
+
+Once you have checked your [Nvidia driver](https://www.nvidia.com/Download/index.aspx) is installed, your Nvidia GPUs on your machine should be [accessible on Livepeer](/video-miners/getting-started/activation#nvidia-gpu-i-ds). 
+ 
+- Use the `nvidia-smi` (installed with the Nvidia driver) to print a list of GPUs:
+
+**For example:**
+
+The following output indicates that there are 5 GPUs accessible on the machine with
+IDs from 0 through 4.
+
+``` bash
+nvidia-smi -L
+GPU 0: GeForce GTX 1070 Ti (UUID: GPU-fcbaffa0-38ae-02d0-5c47-f8fd9922eb75)
+GPU 1: GeForce GTX 1070 Ti (UUID: GPU-d46a085e-0d66-0214-34d3-96860a5c778f)
+GPU 2: GeForce GTX 1070 Ti (UUID: GPU-32b7c120-c2c9-0069-6130-cb67afd89642)
+GPU 3: GeForce GTX 1070 Ti (UUID: GPU-9e4163c3-a120-3cbb-7869-1223b322eab2)
+GPU 4: GeForce GTX 1070 Ti (UUID: GPU-3370975a-f669-e108-6428-602be9eba7d4)
+```
+
+### Start a combined orchestrator and transcoder
+
+You must first start the orchestrator before you activate:
+
+**For example:**
+
+```bash
+livepeer \
+    -network arbitrum-one-mainnet \
+    -ethUrl <ETH_URL> \
+    -ethAcctAddr <ETH_ACCT_ADDR> \ # Only required if you already have an ETH account you want to use
+    -orchestrator \
+    -transcoder \
+    -nvidia <NVIDIA_DEVICE_IDs> \ # Only required for transcoding with Nvidia GPUs
+    -pricePerUnit <PRICE_PER_UNIT> \
+    -serviceAddr <SERVICE_ADDR> # Hostname/IP:port
+```
+
+
+#### Flags
+
+- Use the `-ethAcctAddr` flag to specify the ETH account address that you want the node to use. 
+
+> **Note:** This flag is only required if you already have an account that you want to use.
+
+- Be sure the wallet file for the account address is  stored in the keystore directory. 

```

Comment:
`wallet file` -> `private key`

and can you add a note:
"It's paramount that you securely manage the private key, as it controls your wallet and funds."

Reply:
changing wallet file --> private key throughout the doc unless you have any objection.